README.md

```diff
@@ -1,12 +1,11 @@
-# React + Vite
+# App name and description
+The app is called My todos and displays a list of to-dos.
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+## Installation instructions
+Once I cloned my repository, I typed in the terminal the following:
```

Comment:
If you are cloning your repository with the application already created there is no need for `create vite@latest ./ -- --template react`. This will just create another instance of a new app. Just cd into the created app `todo-list` and run `npm install`